packages/fx-core/src/component/generator/copilotPlugin/helper.ts

```diff
@@ -159,6 +160,12 @@ export async function listOperations(
         operations = operations.filter(
           (operation: string) => !existingOperations.includes(operation)
         );
+        // No extra API can be added
+        if (operations.length == 0) {
+          return err([
+            { type: ApiSpecErrorType.NoSupportedApi, content: ConstantString.NoSupportedApi },
```

Comment:
The content of this stinrg here will likely be changed to later to something like  "No supported API found. Only GET and POST methods with single parameters and no auth are supported." Not sure whether it will impact you or not.